dotCMS/src/main/java/com/dotcms/util/pagination/Paginator.java

```diff
@@ -1,5 +1,6 @@
 package com.dotcms.util.pagination;
 
+import com.dotmarketing.util.PaginatedArrayList;
 import com.liferay.portal.model.User;
 
 import java.util.Collection;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Collection'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)